src/pip/_internal/wheel.py

```diff
@@ -68,15 +68,22 @@ def normpath(src, p):
     return os.path.relpath(src, p).replace(os.path.sep, '/')
 
 
-def rehash(path, blocksize=1 << 20):
-    # type: (str, int) -> Tuple[str, str]
+def hash_file(path, blocksize=1 << 20):
+    # type: (str, int) -> Tuple[Any, str]
     """Return (hash, length) for path using hashlib.sha256()"""
     h = hashlib.sha256()
     length = 0
     with open(path, 'rb') as f:
         for block in read_chunks(f, size=blocksize):
             length += len(block)
             h.update(block)
+    return (h, str(length))  # type: ignore
```

Comment:
This could return the length as `int` since `rehash` uses `str(length)` and `_build_one_inside_env` ignores it

Reply:
Good observation. For that matter (but not for this PR), `rehash()` should probably also be made to return an int! (And the caller can convert to a string as needed.)

Reply:
I have made ``hash_file`` return a int with 53c38d8.  I will leave ``rehash()`` for now to avoid further confusion :)